src/CashFlows.py

```diff
@@ -28,7 +28,9 @@
 
 import numpy as np
 import time
-
+###
+import itertools as iter
```

Comment:
I wouldn't call this `iter` as `iter()` is a built-in function.  I typically call it `it` or `itools`